src/core/dbus-unit.c

```diff
@@ -936,6 +977,9 @@ const sd_bus_vtable bus_unit_vtable[] = {
         SD_BUS_PROPERTY("Refs", "as", property_get_refs, 0, 0),
         SD_BUS_PROPERTY("ActivationDetails", "a(ss)", bus_property_get_activation_details, offsetof(Unit, activation_details), SD_BUS_VTABLE_PROPERTY_EMITS_CHANGE),
         SD_BUS_PROPERTY("DebugInvocation", "b", bus_property_get_bool, offsetof(Unit, debug_invocation), 0),
+        SD_BUS_PROPERTY("StateDirectoryQuotaUsage", "(bbtt)", property_get_quota, 0, SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("CacheDirectoryQuotaUsage", "(bbtt)", property_get_quota, 0, SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("LogsDirectoryQuotaUsage", "(bbtt)", property_get_quota, 0, SD_BUS_VTABLE_PROPERTY_CONST),
```

Comment:
as above, these are properties for ExecContext, not Unit

Reply:
or, hmm, you need to access Manager, so ExecContext is also out of question. In that case, I guess there are two options: either this gets added only to Service, or introduce a new vtable like dbus-quota. Because the concept is exec-bound, hence doesn't apply to timer/path/scope/slice units and hence shouldn't live in generic Unit vtable...

Reply:
see https://github.com/systemd/systemd/pull/35892#discussion_r2088726367